src/utils/hapticFeedbacks.js

```diff
@@ -0,0 +1,7 @@
+/* @flow strict-local */
```

Comment:
I don't think we need this file, given that the implementation is just one line.

IMO, this obscures things more than it helps.

Reply:
I added 'selection' for IOS , because there is no deault given for IPhone 6s and 6s+ in [code](https://github.com/junina-de/react-native-haptic-feedback/blob/7372436c2cd70a77447312f9e47cbc7be1709fa2/ios/RNReactNativeHapticFeedback.m#L50), 
and now It has more than one line, so leaved this file :)